### PR TITLE
Add lint npm script

### PR DESCRIPTION
**Pull request by varanthir**
Test od Added Travis Ci

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
-  },
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true
+        }
+    },
```
Comment (ulfryk) [inline .eslintrc.json]: keep consistent indents

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - "4"
+  - "4.2"
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
+  - "0.10"
+  - "0.8"
+  - "0.6"
+  - "iojs"
```
Comment (ulfryk) [inline .travis.yml]: decide which one is important - maybe ` - "5"` will be enough :)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+	"name": "to-do",
+    "devDependencies": {
+        "eslint": "latest"
```
Comment (ulfryk) [inline package.json]: always choose exact version of your dependencies, NPM is not stable envinronment -> latest can break your code :)

File: .eslintrc.json (hunk 1 above)
Comment (ulfryk) [inline .eslintrc.json]: ??

File: package.json (hunk 3 above)
Comment (ulfryk) [inline package.json]: not fixed 👎

File: .eslintrc.json (hunk 1 above)
Comment (ulfryk) [inline .eslintrc.json]: :-1:

File: package.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+	"name": "to-do",
+    "devDependencies": {
+        "eslint": "2.11.1"
+    },
+
+    "scripts": {
+        "test": "eslint script.js"
+    }
+}
```
Comment (ulfryk) [inline package.json]: :-1:

File: script.js [hunk 5]
```diff
@@ -75,7 +75,7 @@ function getNewNoteFromForm(ui) {
         ui.title.value,
         ui.note.value,
         textToTimestamp(ui.time.value, ui.date.value),
-        ui.hour.checked
+ui.hour.checked
```
Comment (ulfryk) [inline script.js]: I think that it should not pass `eslint` test

File: package.json (hunk 4 above)
Comment (varanthir) [inline package.json]: In my editor, it was ok..

Comment (ulfryk) [inline package.json]: you mixed spaces and tabs - replace tabs with spaces - set your editor to use spaces :)

File: script.js (hunk 5 above)
Comment (varanthir) [inline script.js]: #1801

File: package.json (hunk 4 above)
Comment (varanthir) [inline package.json]: Ok, thx, now I see.

File: script.js (hunk 5 above)
Comment (ulfryk) [inline script.js]: I see.. anyway - please fix it :)

File: package.json (hunk 4 above)
Comment (ulfryk) [inline package.json]: most common indent for `.json` files is 2 spaces

Comment (varanthir) [inline package.json]: Thx

File: script.js (hunk 5 above)
Comment (varanthir) [inline script.js]: Done.

**Comment by varanthir** [comment]
@ulfryk, ok to PR?

**Comment by ulfryk** [comment]
@varanthir - looks good :+1:

**Pull request merged**